HLeithner/joomla-cms
# Add prepared statements for plg-finder-categories

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the plugin in all ways you can think of.

This plugin index the categories in smart search.

* Activate smart search
* Create categories
* Change the category text
* Delete categories
* Check the index of smart search in the backend and frontend or tables-


### Expected result
Nothing changed.
